assimilation_code/modules/utilities/no_cray_winf08_mod.f90

```diff
@@ -82,20 +77,15 @@ subroutine create_state_window(state_ens_handle, fwd_op_ens_handle, qc_ens_handl
    my_num_vars = state_ens_handle%my_num_vars
 
    call mpi_type_size(datasize, bytesize, ierr)
-   window_size = my_num_vars*data_count*bytesize
-
-   allocate(contiguous_fwd(data_count*my_num_vars))
-   contiguous_fwd = reshape(state_ens_handle%copies(1:data_count, :), (/my_num_vars*data_count/))
+   window_size = my_num_vars*state_ens_handle%num_copies*bytesize
```

Comment:
```suggestion
   window_size = my_num_vars*data_count*bytesize
```
If we don't want to use data_count here and instead want to use state_ens_handle%num_copies, then we don't need the data_count variable at all. This contradicts the comment on line 64 though

Reply:
data_count is the number of copies that get_state will get, state_ens_handle%num_copies is all the copies
![Screenshot 2024-10-31 at 10 56 02 AM](https://github.com/user-attachments/assets/b21a44f8-83c1-40b8-ae71-c9f40f7a6493)